l2beat/l2beat
# Prune zero-value reports

Resolves L2B-2088

## What's changed

- No more zeroed values across repots
- Detailed controller is aware of potential data gaps due to lack of zeroed reports and fills the timeframes beforehand
- Data migration to get rid of zeroed values across reports
- insert filtering for zeroed reports

## How to test it?
Run staging dump locally and start local backend and frontend local variant. Data should be slim, charts will work as if no changes were made
